doc/explanation/storage.md

```diff
@@ -154,7 +154,7 @@ Each storage volume uses one of the following content types:
   You can create a custom storage volume of type `block` by using the `--type=block` flag.
 
   Custom storage volumes of content type `block` can only be attached to virtual machines.
-  They should not be shared between instances, because simultaneous access can lead to data corruption.
+  By default they can only be attached to one instance at a time, because simultaneous access can lead to data corruption.
```

Comment:
@ru-fu should this section mention that this behaviour can be changed using `security.shared`?

Reply:
Also, just thought about this, but should I mention unshared block volumes can't be attached do profiles? And if so which section would be more appropriate?

Reply:
Just pushed some changes adding a mention to profiles [here](https://github.com/canonical/lxd/pull/13183/commits/ea842ab5afd3fb94f0e5dc4e1103f85aedcec144#diff-5dbfa72d6725876d0859b18af2b778bee3a40fe7669b14c19b5485c327abb0c7R56).

Reply:
```suggestion
  By default, they can only be attached to one instance at a time, because simultaneous access can lead to data corruption.
  Set the `security.shared` storage option to allow sharing volumes between instances.
```

Reply:
Sorry, missed your comment Tom! But yes, adding that here makes sense. Unfortunately we can't link to the option since it's specific to the storage driver. :(

Reply:
Regarding the profiles, I don't think it's needed here (because it's implied by saying they can only be attached to one instance). So it makes sense to add it to the how-to like you did.